### ISSUE TITLE
Audio offset bug after lag makes the game unplayable

### ISSUE THREAD

**Issue by Packsolite** [ppy/osu#28879]
### Type

Game behaviour

### Bug description

There is a weird lag at the start of every beatmap that causes the audio track to be offset.

### Screenshots or videos

Notice the short freeze in the beginning of the video.
https://youtu.be/IjwatJqvomc

### Version

2024.625.2-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16241930/compressed-logs.zip)

**Comment by smoogipoo** [comment]
Very weird. You say it's with that program running though I can repro to a much smaller/mostly unnoticeable degree without it.

**Comment by Packsolite** [comment]
> Very weird. You say it's with that program running though I can repro to a much smaller/mostly unnoticeable degree without it.

Yeah, the lag is always there, but it's barely noticable usually. But with the monitoring software running, the game is literally unplayable.

**Comment by PasswordOnOsu** [comment]
I had the same issue. What solved it for me was turning off the audio monitoring on voicemeeter for every option. For some reason it interfered with the syncing.

**Comment by PasswordOnOsu** [comment]
> I had the same issue. What solved it for me was turning off the audio monitoring on voicemeeter for every option. For some reason it interfered with the syncing.

it seems like being in a discord call could stop this solution from working? otherwise it works fine when im not in a call from what i can tell.

**Comment by Vexcited** [comment]
Happens to me without having any third party software running in background, not sure if there's any hidden background system service that might cause this though. Played osu!lazer on Windows and Linux (Arch) and both OSes caused this audio offset...

**Comment by Packsolite** [comment]
Seems like BTMC has the same problem in his streams: https://youtu.be/Snn9ZNt9WTQ?t=375

**Comment by peppy** [comment]
It's not the same problem. BTMC found out he had a wack global offset.

**Comment by Packsolite** [comment]
> It's not the same problem. BTMC found out he had a wack global offset.

You sure about that? He later mentioned that he closed some application "real.exe" and it fixed it...

**Comment by peppy** [comment]
> You sure about that? He later mentioned that he closed some application "real.exe" and it fixed it. It seems like he put the "wack global offset" because of that issue...

even more sure now. yes

**Comment by peppy** [comment]
Are there any improvements to this in the latest release? We updated some audio libraries which may have made this better or worse.

**Comment by peppy** [comment]
At that point you just need to close the software. It's in no way our fault.
